libvmi/accessors.c

```diff
@@ -467,3 +467,22 @@ addr_t vmi_translate_uv2p (vmi_instance_t vmi, addr_t virt_address, vmi_pid_t pi
     }
     return paddr;
 }
+
+const char *
+vmi_get_linux_sysmap(
+	vmi_instance_t vmi)
+{
+    linux_instance_t linux_instance = NULL;
+
+    if(VMI_OS_LINUX != vmi->os_type || (VMI_INIT_PARTIAL & vmi->init_mode))
```

Comment:
Nit: could you put curly brackets around this if clause (like you do in the following one)?